modules/core/test/classes/matrix4.spec.ts

```diff
@@ -42,6 +42,11 @@ test('Matrix4#fromQuaternion', (t) => {
   t.end();
 });
 
+test('Matrix4#fromTranslationQuaternion', (t) => {
+  tapeEquals(t, new Matrix4().fromTranslationQuaternion([0, 0, 0], [0, 0, 0, 1]), IDENTITY_MATRIX);
```

Comment:
Appreciate the test case. Maybe add one test case with non-zero vectors to test the non-trivial case?

Reply:
Yes, a litlle bit lazy here !!! Just inspired myself from the test case Matrix4#fromQuaternion. I'll do my best to update that !

Reply:
Thanks, admit that our existing tests are not always that good. But that seems like a poor reason for keeping a bad habit in new code....

Reply:
Done ! Tell me what you think